low-level/files/opt/seagate/sspl/setup/sspl_test.py

```diff
@@ -148,7 +152,8 @@ def process(self):
             # TODO: Convert shell script to python
             # from cortx.sspl.sspl_test.run_qa_test import RunQATest
             # RunQATest(self.plan, self.avoid_rmq).run()
-            CMD = "%s/run_qa_test.sh %s %s" % (TEST_DIR, self.plan, self.avoid_rmq)
+            CMD = "%s/run_qa_test.sh %s %s %s"\
+                   %(TEST_DIR, self.plan, self.avoid_rmq, self.coverage_enabled)
```

Comment:
what is this avoid_rmq, why we are still dealing with rmq? need to do away with it. Update all relevant documentation after this.
Also change arguments to kv pairs, so position wont matter when we add/remove any argument

Reply:
romved avoid_rmq and added kv args to run_qa_test.sh. 👍